packages/react-components/src/theming/FluentThemeProvider.tsx

```diff
@@ -14,6 +14,15 @@ export interface FluentThemeProviderProps {
   children: React.ReactNode;
   /** Theme for components. Defaults to a light theme if not provided. */
   fluentTheme?: PartialTheme | Theme;
+  /**
+   * Defines where body-related theme is applied to.
+   * Setting to 'element' will apply body styles to the root element of ThemeProvider.
+   * Setting to 'body' will apply body styles to document body.
+   * Setting to 'none' will not apply body styles to either element or body.
+   *
+   * @defaultvalue 'element'
+   */
+  applyTo?: 'element' | 'body' | 'none';
```

Comment:
instead of manually defining this, is there a type exposed for this we could use from the ThemeProvider's package? I can't find one so very possible there isn't

Reply:
There is not unfortunately. I basically just copied their comment and typing.
![image](https://user-images.githubusercontent.com/79475487/128082756-e82a3465-141d-40e6-a1c8-4bdaf8e01ba9.png)

Reply:
Thanks for checking this! Food for thought (not in this PR): should we avoid inline string literal definitions and instead export them as types so they can be imported and used by customers?

Reply:
That is a good point but I have now created a new PR for a better fix here https://github.com/Azure/communication-ui-library/pull/641. Sorry for the sudden change but thank you for reviewing. I will close this PR.